packages/MarkdownEditor-Core.package/MarkdownToken.class/class/newFor..st

```diff
@@ -0,0 +1,7 @@
+instance creation
+newFor: aMarkdownReadStream
+
+	^ self basicNew 
+		initialize: aMarkdownReadStream;
+		position: aMarkdownReadStream position + 1;
```

Comment:
No request for change, but I hate the way Squeak is handling streams and positions. 🙄😂